### PR TITLE
VIH-6957 interpreter domain changes

### PR DESCRIPTION
**Pull request by phillipjchambers**
### JIRA link (if applicable) ###

VIH-6957

### Change description ###

Add interpreter/linked participant support to:
- AddParticipantsToConferenceCommand (ParticipantController)
- CreateConferenceCommand (ConferenceController)

Includes:
- update requests & responses
- add DTO
- add a new exception - ParticipantLinkException
- add/update mappings
- add tests
- add DB migration

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: VideoApi/VideoApi.DAL/Exceptions/ConferenceNotFoundException.cs [hunk 1]
```diff
@@ -1,15 +1,22 @@
 using System;
+using System.Runtime.Serialization;
 
 namespace VideoApi.DAL.Exceptions
 {
-#pragma warning disable S3925 // "ISerializable" should be implemented correctly
+    [Serializable]
```
Comment (shaed-parkar) [inline VideoApi/VideoApi.DAL/Exceptions/ConferenceNotFoundException.cs:6]: Thanks for this!

**Review by shaed-parkar — APPROVED** [review]
LGTM. Good work!

File: VideoApi/VideoApi.DAL/Commands/CreateConferenceCommand.cs [hunk 2]
```diff
@@ -58,6 +64,24 @@ public async Task Handle(CreateConferenceCommand command)
             {
                 conference.AddParticipant(participant);
             }
```
Comment (williambowshersolirius) [inline VideoApi/VideoApi.DAL/Commands/CreateConferenceCommand.cs:66]: Any reason why this code block is not moved to another class, as its exactly the same in the AddParticipantsToConferenceCommand?

**Review by williambowshersolirius — APPROVED** [review]
All looks good, just 1 set of duplicate code between the Commands

**Review by robin-weller — DISMISSED** [review]

**Review by shaed-parkar — APPROVED** [review]

**Pull request merged**